Makefile

```diff
@@ -1,22 +1,23 @@
-default: build/universal-robots
+.PHONY: module.tar.gz ur5e-sim format test clean clean-all docker docker-build docker-amd64 docker-upload appimages docker-arm64-ci docker-amd64-ci
+default: format build/universal-robots
 
 # format the source code
 format:
 	ls src/*.*pp main.cpp | xargs clang-format-15 -i --style=file
 
 build: 
-	mkdir build logs
+	mkdir build
 
 test: build/universal-robots
 	./build/universal-robots-test
 
-build/universal-robots: format build
```

Comment:
can we change this back? with the current setup the appimage builds will not call `format`. its not the end of the world if I have to call format again tho

Reply:
we can't run format as format isn't always installed on the host machine, that said I'll add format to all the targets that currently depend on it